### ISSUE TITLE
[css-text] Specify HTML alignment as text-align: -webkit-{left,right,center}

### ISSUE THREAD

**Issue by emilio** [w3c/csswg-drafts#10388]
All browser engines implement [the HTML align attribute](https://html.spec.whatwg.org/#flow-content-3:attr-div-align) with a special value of `text-align`, rather than as the HTML spec describes. Those values are `-{moz,webkit}-{left,right,center}`, and for mostly legacy accidents, they are exposed to stylesheets and actively used (yikes).

Rather than keeping the status quo where the specification doesn't contain these values, and the HTML spec says something that no browser does, I propose to define how these works in terms of the `-webkit-` prefixed version.

Is there any reason not to do that? cc @bfgeek @frivoal

**Comment by bfgeek** [comment]
Yeah if you are having to ship it then we should spec it, and seems resonable.

Arguably this would go in the align spec, as it is very similar to justify-self:blah for block layout (even though nobody has implemented that yet afaik).
https://drafts.csswg.org/css-align-3/#justify-block

**Comment by Loirooriol** [comment]
CC @fantasai as per https://github.com/whatwg/html/issues/3428

**Comment by nicoburns** [comment]
> <TabAtkins> dbaron: Because for example you can override this inherited behavior by setting another value on text-align. We need to see if that's something that's ok to break.

My suspicion is that this will turn out not to be something that's ok. The pattern which (with my web author's hat on) I suspect to be common is:

```html
<center>
    <div style="text-align: left">...</div>
</center>
```
